### PR TITLE
chore: bump tracing-subscriber to 0.3

### PR DESCRIPTION
**Pull request by PhotonQuantum**

### CODE REVIEW

**Comment by seanpianka** [comment]
FYI: this is a breaking change for downstream users of the crate.

**Comment by PhotonQuantum** [comment]
How about bumping the major version, following the upstream?

**Comment by dbrgn** [comment]
> How about bumping the major version, following the upstream?

Yes, I'll definitely bump the version 🙂 I'll try to take a look at the PR soon, today or tomorrow.

**Comment by nmathewson** [comment]
Friendly ping: this would be extremely helpful, if and when you have time to look at it.

**Comment by dbrgn** [comment]
Thanks for the reminder, I forgot about this!

File: tracing-test/Cargo.toml [hunk 1]
```diff
@@ -13,12 +13,12 @@ categories = ["development-tools::testing"]
 [dependencies]
 lazy_static = "1.4"
 tracing-core = "0.1"
-tracing-subscriber = "0.2"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
```
Comment (dbrgn) [inline tracing-test/Cargo.toml:16]: The `env-filter` feature is already enabled by default, but it doesn't hurt to explicitly list it!

**Review by dbrgn — APPROVED** [review]
Thank you, @PhotonQuantum!

File: tracing-test/Cargo.toml (hunk 1 above)
Comment (PhotonQuantum) [reply tracing-test/Cargo.toml:16]: It's actually no longer enabled by default for tracing_subscriber 0.3 ;)
ref: https://docs.rs/crate/tracing-subscriber/0.3.2/features

Comment (dbrgn) [reply tracing-test/Cargo.toml:16]: Oops, thanks for the correction, I accidentally looked at the outdated docs.rs page for 0.2 🙂

**Comment by dbrgn** [comment]
Version 0.2 is released. Note: I don't want to track upstream versions _yet_, because there might be breaking releases in the future independent of `tracing-subscriber`.

**Pull request merged**